### PR TITLE
Add 10  new interns

### PR DESCRIPTION
**Pull request by Ammar000x**
new Appgain, Kazyon-Plus, Xceed, J&T Express, PayMob, Educatly, Kijamii, GMind, Envision, new orange

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,10 +46,13 @@ Stack Info allows you to explore and find tech stacks used by startups and corpo
 |[Dell](https://jobs.dell.com/job/-/-/375/31463452720?source=rd_LinkedIn&utm_source=linkedin.com&utm_medium=job_posting&utm_campaign=various&utm_content=global&utm_term=239560378&ss=paid&dclid=CjkKEQjwwdWVBhD9qde_596T-5MBEiQAIZRjpEDRBLMaw4gaw6XEco4Rg8StxlNNgUNjTO3UqqtRFWLw_wcB)| Cairo |  Open - Undergraduate Technical Support Intern|
 |[Emarat Al-Ard reseach Insitute](https://docs.google.com/forms/d/e/1FAIpQLSf7baCk2wffDEohoQDo41zGTCRMAZloUuH0g-bVtwlxJYdATA/viewform) | Remote | Open - <b>Unpaid Intern</b> - Data Analysts Intern |
 |[E-num](https://www.linkedin.com/posts/hassanel-seoudy_hiring-activity-6938900459000602624--6ew/?utm_source=linkedin_share&utm_medium=ios_app&fbclid=IwAR23uapqZdAM2tKNz0gpKKV-2ahTbF4wA7TGCpWDIu3LV8x3Oq4vaMi52K0) | Remote | Open |
+|[Educatly](https://cutt.ly/Educatly) | Cairo | Open - Deadline to apply: UnKnown|
```
Comment (moaz-eldefrawy) [inline README.md:49]: Educatly internship deadline has passed. Can you remove it?

**Comment by moaz-eldefrawy** [comment]
Thanks a lot for your contributions, Ammar ❤️. I have added some notes. Feel free to let me know if you want me to make these changes. Thanks again ❤️.

File: README.md [hunk 2]
```diff
@@ -44,10 +46,13 @@ Stack Info allows you to explore and find tech stacks used by startups and corpo
...
 |[Enozom](https://www.enozom.com/) | Alexandria | Not Yet |
+|[Envision](https://cutt.ly/envisionemployment) | 6TH OF OCTOBER, GIZA | Open - Deadline to apply: UnKnown|
 |[Espace](https://espace.com.eg/blog/announcing-our-2022-internship-program/) | Remote |Open - <b>Paid Intern</b> - Deadline: End of June|
 |[Gameball](https://www.facebook.com/Gameball.co/photos/a.339279460114806/999153637460715/) | Cairo | Open - <b>Kickstart program</b>|
 |[Giza Systems](https://www.gizasystemscareers.com/en/egypt/jobs/headway-intern-2022-software-track-4512285/) | Cairo | Open |
+|[GMindJ](https://cutt.ly/GmindJobs) | Remotely | Open - Deadline to apply: UnKnown|
```
[the remark this answers is not in the document]
Comment (Ammar000x) [reply README.md:56]: Why irrelevant?
In Kijamii there is (FrontEnd & BackEnd) in the form link 
J&T Express (not sure what is this)
PayMob (seems reasonable to me)
Alexandria For Programming ( normal internship in wuzzuf)
Envision (seems official from their site)

and the links are only shortened as when I edit the code, I'm not too fond of the link being very long as I post links from Facebook and LinkedIn

**Comment by Ammar000x** [comment]
> Thanks a lot for your contributions, Ammar ❤️. I have added some notes. Feel free to let me know if you want me to make these changes. Thanks again ❤️.

you are welcome, Make what is needed to be done.
I am only helping to find some internships for colleagues.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (moaz-eldefrawy) [reply README.md:55]: I meant the GMindJ internship in line 55. It has many internships and only [one](https://gmindeg.com/job/android-developer/) that's relevant.

**Pull request merged**